# 机器人位置消息
# 包含6个机器人的坐标，每个坐标由x和y组成
float32[2] position_1_up
float32[2] position_2_up
float32[2] position_3_up
float32[2] position_4_up
float32[2] position_7_up
bool[5] if_guess